website/content/docs/audit/index.mdx

```diff
@@ -89,6 +89,57 @@ using the [secrets enable](/vault/docs/commands/secrets/enable) command, and the
 
 [auth tune](/vault/docs/commands/auth/tune)
 
+## Audit request headers
+
+Vault Operators can audit specific HTTP headers when those headers are present in
+the incoming client request. Configuration for audit request headers is applied via
+the [Vault API](/vault/api-docs/system/config-auditing).
+
+When Vault Operators [create](/vault/api-docs/system/config-auditing#create-update-audit-request-header)
+an exception allowing a request header to appear in the audit log, by default the
+value of the header is **not** HMAC'd, this is [configurable](/vault/api-docs/system/config-auditing#hmac)
+per header.
+
+### Default headers
+
+<Warning title="Behavior changes">
+
+  Prior to Vault 1.17 **only** headers configured by the Vault Operator could appear
+  in an audit log.
+
+  As of Vault 1.17 this behavior changes, see below for further information.
+
+</Warning>
+
+As of Vault 1.17 the following headers are **always automatically enabled** in
+order to facilitate correlation of requests across distributed systems which
+include Vault.
```

Comment:
```suggestion
To help correlate requests across distributed systems, Vault automatically
records the following headers in the audit log:
```
Style correction: write in active voice

Reply:
In other documents on our site we call out 'since Vault `x.xx` to let folks know that it changed then, I think that seems useful to know. What do you recommend in order to capture this info for readers?

Reply:
I know the current docs do that a lot, it was an established pattern prior to versioning. I've been trying to remove it as I encounter it since readers should be relying on the version selector to know if a feature is applicable to their version.